examples/simple.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"log"
 
-	_ "github.com/exasol/go-exasol"
+	_ "github.com/exasol/exasol-driver-go"
 )
 
 func main2() {
```

Comment:
Maybe can you rename the method to `main`

Reply:
I wanted to do it in the next pull request, I have a ticket for that: https://github.com/exasol/exasol-driver-go/issues/9